### PR TITLE
Update download icons

### PR DESCRIPTION
**Pull request by dimasciput**
Hi @sumandari , can you test my PR? Thanks

### CODE REVIEW

**Comment by sumandari** [comment]
@dimasciput 

This is how it looks on my local. Cool!
image

File: django_project/lesson/static/css/worksheet_detail.css [hunk 1]
```diff
@@ -19,3 +19,7 @@ Inspired from the "details" class in organisation_detail.css
     margin: 0 0 10px;
     font-size: 0.75em;
 }
+
+.btn-mini {
+    height: 3em !important;
+}
```
Comment (sumandari) [inline django_project/lesson/static/css/worksheet_detail.css]: for bigger glyphicon, we can apply class `gi-big`:
```
.gi-big {
    font-size: 1.5em !important;
}
```

Comment (sumandari) [reply django_project/lesson/static/css/worksheet_detail.css]: e.g if we apply it on back-arrow icon
image

**Review by sumandari — APPROVED** [review]

**Comment by timlinux** [comment]
Could we keep these monochrome to be more consistent with the rest of the site? Thanks!

**Comment by sumandari** [comment]
@timlinux @dimasciput 
image

**Pull request merged**